atuin-client/Cargo.toml

```diff
@@ -54,6 +54,7 @@ futures = "0.3"
 crypto_secretbox = "0.1.1"
 generic-array = { version = "0.14", features = ["serde"] }
 serde_with = "3.5.1"
+ratatui = { workspace = true }
```

Comment:
I'm not a _huge_ fan of this overall, but I think it's ok for now. Totally out of scope for this PR, but I'd like to reorg our crates a bit anyway.